plugins/pulp_python/plugins/models.py

```diff
@@ -18,7 +18,8 @@ class Package(object):
     # The full list of supported attributes. Attributes beginning with underscore are specific to
     # this module and are not found in PKG-INFO.
     _ATTRS = ('name', 'version', 'summary', 'home_page', 'author', 'author_email', 'license',
-              'description', 'platform', '_filename', '_checksum', '_checksum_type')
+              'description', 'platform', '_filename', '_checksum', '_checksum_type',
+              '_metadata_file')
```

Comment:
I'm curious - why store the metadata file here? It's not harming anything, but I am wondering what use case it is needed for.